test/utils.py

```diff
@@ -39,13 +39,13 @@ def setup_test_logging(logger: logging.Logger, filename: str):
     )
     formatter = logging.Formatter(log_fmt)
 
-    if os.getenv("STREAM_LOG", "true"):
+    if os.getenv("STREAM_LOG", "true").lower() == "true":
```

Comment:
**Problem:**
When `STREAM_LOG=False`, it previously evaluated to `True` because `bool("False") == True`